workitem_blackbox_test.go

```diff
@@ -736,10 +748,41 @@ func (s *WorkItem2Suite) TestWI2UpdateMultipleScenarios() {
 	s.minimumPayload.Data.Attributes["version"] = updatedWI.Data.Attributes["version"]
 }
 
-func (s *WorkItem2Suite) TestWI2SuccessCreateWorkItem() {
+// TestWI2SuccessCreateWorkItemWithDescription verifies that the `workitem.SystemDescription` attribute is not set, as well as its pair workitem.SystemDescriptionMarkup when the work item description is not provided
+func (s *WorkItem2Suite) TestWI2SuccessCreateWorkItemWithoutDescription() {
+	c := minimumRequiredCreatePayload()
+	c.Data.Attributes[workitem.SystemTitle] = "Title"
+	c.Data.Attributes[workitem.SystemState] = workitem.SystemStateNew
+	c.Data.Relationships = &app.WorkItemRelationships{
+		BaseType: &app.RelationBaseType{
+			Data: &app.BaseTypeData{
+				Type: "workitemtypes",
+				ID:   "system.bug",
+			},
+		},
+	}
+
+	_, wi := test.CreateWorkitemCreated(s.T(), s.svc.Context, s.svc, s.wi2Ctrl, &c)
+	assert.NotNil(s.T(), wi.Data)
```

Comment:
I generally prefer require because then I get the first error at a glance so feel free to choose. The only hard requirement (I can think of right now) is to use `require` if you first check for `NotNil` and then dereference the tested pointer.